nextflow.config

```
@@ -65,6 +65,12 @@ params {
 
     // kraken2
     run_kraken2                = false
+
+    // centrifuge
+    run_centrifuge             = false
+    save_unaligned             = false
```

```suggestion
    save_unaligned             = false
```

For all these parameters can you please append with `centrifuge_` so we know which tool it's relevant for.